test/integration/controller/admissionchecks/provisioning/provisioning_test.go

```diff
@@ -592,15 +597,32 @@ var _ = ginkgo.Describe("Provisioning", ginkgo.Ordered, ginkgo.ContinueOnFailure
 			})
 
 			ginkgo.By("Checking the admission check is ready", func() {
+				updatedWl := &kueue.Workload{}
 				gomega.Eventually(func(g gomega.Gomega) {
 					g.Expect(k8sClient.Get(ctx, wlKey, updatedWl)).To(gomega.Succeed())
 					state := workload.FindAdmissionCheck(updatedWl.Status.AdmissionChecks, ac.Name)
 					g.Expect(state).NotTo(gomega.BeNil())
 					g.Expect(state.State).To(gomega.Equal(kueue.CheckStateReady))
+					g.Expect(state.PodSetUpdates).To(gomega.BeComparableTo([]kueue.PodSetUpdate{
+						{
+							Name: "ps1",
+							Annotations: map[string]string{
+								provisioning.ConsumesAnnotationKey: provReqKey.Name,
+							},
+						},
+						{
+							Name: "ps2",
+							Annotations: map[string]string{
+								provisioning.ConsumesAnnotationKey: provReqKey.Name,
+							},
+						},
+					}))
 				}, util.Timeout, util.Interval).Should(gomega.Succeed())
 			})
 
 			ginkgo.By("Deleting the provision request", func() {
+				createdRequest := &autoscaling.ProvisioningRequest{}
+				createdRequest.Name, createdRequest.Namespace = provReqKey.Name, provReqKey.Namespace
```

Comment:
Instead of generating an object, should we get a PR object?

Reply:
This is better, as it avoids an API call. However, prefer using this style:
```
object := &Type{
  field: value
}
```

Reply:
I'm ok with @alculquicondor recommendation way.

Reply:
Done.